show/main.py

```diff
@@ -3050,8 +3050,8 @@ def neighbors():
                 for ip, mac in nbrs_data[intf]:
                     r = ["", ip, mac, intf]
                     table.append(r)
-        click.echo(tabulate(table, header))
-        click.echo("\n")
+    click.echo(tabulate(table, header))
```

Comment:
we should keep the click.echo() in side the for loop, since the output is grouped via vnet_name. if outside, only the last vnet is shown.
e.g.

 ```
  admin@sonic:~$ show vnet neighbors
  Vnet_2000    neighbor     mac_address    interfaces
  -----------  -----------  -------------  ------------
               11.11.11.11                 Ethernet1
               11.11.11.12                 Ethernet1
  Vnet_3000    neighbor     mac_address        interfaces
  -----------  -----------  -----------------  ------------
               20.20.20.20  aa:bb:cc:dd:ee:ff  Vlan2000
  ```

For the empty case , we can add a flag like shown = False by default, and if in the for loop, set it to true, and if false ouside the for loop, show the empty case.
e.g.

    ```
    shown = False
    table = []
    for k, v in vnet_intfs.items():
        v = natsorted(v)
        header[0] = k
        table = []
        for intf in v:
            if intf in nbrs_data:
                for ip, mac in nbrs_data[intf]:
                    r = ["", ip, mac, intf]
                    table.append(r)
        shown = True
        click.echo(tabulate(table, header))
        click.echo("\n")
    
    if not shown:
        click.echo(tabulate(table, header))
```

Reply:
Thanks @dzhangalibaba, i just added a check to see if `vnet_intfs `is empty and display the default header. Mostly the same logic as you suggested. Can you check?

Reply:
> Thanks @dzhangalibaba, i just added a check to see if `vnet_intfs `is empty and display the default header. Mostly the same logic as you suggested. Can you check?

LGTM